Playground fails to launch from repo root

### Prerequisites

- [X] I have searched the existing issues
- [X] I understand that providing a [SSCCE](http://sscce.org/) example is tremendously useful to the maintainers.
- [X] I have read the [documentation](https://rjsf-team.github.io/react-jsonschema-form/docs)
- [ ] Ideally, I'm providing a [sample JSFiddle](https://jsfiddle.net/n1k0/f2y3fq7L/6/) or a [shared playground link](https://rjsf-team.github.io/react-jsonschema-form/) demonstrating the issue.

### What theme are you using?

core

### Version

5.2.0

### Current Behavior

Attempting to run `npm start` from the repo root results in a blank screen and a vite error about failing to resolve entry for `@rjsf/material-ui`:

![image](https://user-images.githubusercontent.com/11481499/223245720-0748c314-75f2-4fe0-9bf8-39efa3424ee1.png)

This is coupled with a lot of errors in the console, resulting a completely unusable playground and an inability to develop a pull request for it.

Launching via `npm start` from the `packages/playground` folder works without trouble but doesn't seem to support live-reloading or any sort of output via console from the `packages/core` library.

The output:

```
$ npm start

> start
> lerna run start --stream --parallel

lerna notice cli v6.4.1
lerna WARN run "parallel", "sort", and "no-sort" are ignored when Nx targets are configured. See https://lerna.js.org/docs/lerna6-obsolete-options for details.

 >  Lerna (powered by Nx)   Running target start for 9 projects:

    - @rjsf/antd
    - @rjsf/bootstrap-4
    - @rjsf/chakra-ui
    - @rjsf/core
    - docs
    - @rjsf/fluent-ui
    - @rjsf/material-ui
    - @rjsf/mui
    - @rjsf/playground

 ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————

> @rjsf/core:start


> @rjsf/antd:start


> @rjsf/bootstrap-4:start


> @rjsf/chakra-ui:start


> @rjsf/fluent-ui:start


> @rjsf/material-ui:start


> @rjsf/mui:start


> docs:start


> @rjsf/playground:start

@rjsf/core: > @rjsf/core@5.2.0 start
@rjsf/core: > dts watch
@rjsf/antd: > @rjsf/antd@5.2.0 start
@rjsf/antd: > dts watch
@rjsf/bootstrap-4: > @rjsf/bootstrap-4@5.2.0 start
@rjsf/bootstrap-4: > dts watch
@rjsf/fluent-ui: > @rjsf/fluent-ui@5.2.0 start
@rjsf/fluent-ui: > dts watch
@rjsf/chakra-ui: > @rjsf/chakra-ui@5.2.0 start
@rjsf/chakra-ui: > dts watch
docs: > docs@5.2.0 start
docs: > docusaurus start
@rjsf/mui: > @rjsf/mui@5.2.0 start
@rjsf/mui: > dts watch
@rjsf/material-ui: > @rjsf/material-ui@5.2.0 start
@rjsf/material-ui: > dts watch
@rjsf/playground: > @rjsf/playground@5.2.0 start
@rjsf/playground: > vite --force
@rjsf/playground: Forced re-optimization of dependencies
@rjsf/playground:   VITE v4.1.1  ready in 604 ms
@rjsf/playground:   ➜  Local:   http://localhost:8080/
@rjsf/playground:   ➜  Network: use --host to expose
@rjsf/playground:   ➜  press h to show help
@rjsf/playground: ✘ [ERROR] [plugin vite:dep-scan] Failed to resolve entry for package "@rjsf/material-ui". The package may have incorrect main/module/exports specified in its package.json.
@rjsf/playground:     node_modules/esbuild/lib/main.js:1365:21:
@rjsf/playground:       1365 │         let result = await callback({
@rjsf/playground:            ╵                      ^
@rjsf/playground:     at packageEntryFailure (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22004:11)
@rjsf/playground:     at resolvePackageEntry (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22001:5)
@rjsf/playground:     at tryNodeResolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21736:20)
@rjsf/playground:     at Context.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21487:28)
@rjsf/playground:     at Object.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41587:46)
@rjsf/playground:     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
@rjsf/playground:     at async resolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41815:26)
@rjsf/playground:     at async file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41989:34
@rjsf/playground:     at async requestCallbacks.on-resolve (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1365:22)
@rjsf/playground:     at async handleRequest (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:727:13)
@rjsf/playground:   This error came from the "onResolve" callback registered here:
@rjsf/playground:     node_modules/esbuild/lib/main.js:1287:20:
@rjsf/playground:       1287 │       let promise = setup({
@rjsf/playground:            ╵                     ^
@rjsf/playground:     at setup (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41979:19)
@rjsf/playground:     at handlePlugins (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1287:21)
@rjsf/playground:     at buildOrServeImpl (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:974:5)
@rjsf/playground:     at Object.buildOrServe (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:780:5)
@rjsf/playground:     at /home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:2132:17
@rjsf/playground:     at new Promise (<anonymous>)
@rjsf/playground:     at Object.build (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:2131:14)
@rjsf/playground:     at build (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1978:51)
@rjsf/playground:     at scanImports (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41760:11)
@rjsf/playground:   The plugin "vite:dep-scan" was triggered by this import
@rjsf/playground:     src/index.jsx:2:36:
@rjsf/playground:       2 │ import { Theme as MuiV4Theme } from "@rjsf/material-ui";
@rjsf/playground:         ╵                                     ~~~~~~~~~~~~~~~~~~~
@rjsf/playground: ✘ [ERROR] [plugin vite:dep-scan] Failed to resolve entry for package "@rjsf/mui". The package may have incorrect main/module/exports specified in its package.json.
@rjsf/playground:     node_modules/esbuild/lib/main.js:1365:21:
@rjsf/playground:       1365 │         let result = await callback({
@rjsf/playground:            ╵                      ^
@rjsf/playground:     at packageEntryFailure (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22004:11)
@rjsf/playground:     at resolvePackageEntry (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22001:5)
@rjsf/playground:     at tryNodeResolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21736:20)
@rjsf/playground:     at Context.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21487:28)
@rjsf/playground:     at Object.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41587:46)
@rjsf/playground:     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
@rjsf/playground:     at async resolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41815:26)
@rjsf/playground:     at async file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41989:34
@rjsf/playground:     at async requestCallbacks.on-resolve (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1365:22)
@rjsf/playground:     at async handleRequest (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:727:13)
@rjsf/playground:   This error came from the "onResolve" callback registered here:
@rjsf/playground:     node_modules/esbuild/lib/main.js:1287:20:
@rjsf/playground:       1287 │       let promise = setup({
@rjsf/playground:            ╵                     ^
@rjsf/playground:     at setup (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41979:19)
@rjsf/playground:     at handlePlugins (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1287:21)
@rjsf/playground:     at buildOrServeImpl (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:974:5)
@rjsf/playground:     at Object.buildOrServe (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:780:5)
@rjsf/playground:     at /home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:2132:17
@rjsf/playground:     at new Promise (<anonymous>)
@rjsf/playground:     at Object.build (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:2131:14)
@rjsf/playground:     at build (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1978:51)
@rjsf/playground:     at scanImports (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41760:11)
@rjsf/playground:   The plugin "vite:dep-scan" was triggered by this import
@rjsf/playground:     src/index.jsx:3:36:
@rjsf/playground:       3 │ import { Theme as MuiV5Theme } from "@rjsf/mui";
@rjsf/playground:         ╵                                     ~~~~~~~~~~~
@rjsf/playground: ✘ [ERROR] [plugin vite:dep-scan] Failed to resolve entry for package "@rjsf/fluent-ui". The package may have incorrect main/module/exports specified in its package.json.
@rjsf/playground:     node_modules/esbuild/lib/main.js:1365:21:
@rjsf/playground:       1365 │         let result = await callback({
@rjsf/playground:            ╵                      ^
@rjsf/playground:     at packageEntryFailure (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22004:11)
@rjsf/playground:     at resolvePackageEntry (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22001:5)
@rjsf/playground:     at tryNodeResolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21736:20)
@rjsf/playground:     at Context.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21487:28)
@rjsf/playground:     at Object.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41587:46)
@rjsf/playground:     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
@rjsf/playground:     at async resolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41815:26)
@rjsf/playground:     at async file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41989:34
@rjsf/playground:     at async requestCallbacks.on-resolve (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1365:22)
@rjsf/playground:     at async handleRequest (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:727:13)
@rjsf/playground:   This error came from the "onResolve" callback registered here:
@rjsf/playground:     node_modules/esbuild/lib/main.js:1287:20:
@rjsf/playground:       1287 │       let promise = setup({
@rjsf/playground:            ╵                     ^
@rjsf/playground:     at setup (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41979:19)
@rjsf/playground:     at handlePlugins (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1287:21)
@rjsf/playground:     at buildOrServeImpl (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:974:5)
@rjsf/playground:     at Object.buildOrServe (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:780:5)
@rjsf/playground:     at /home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:2132:17
@rjsf/playground:     at new Promise (<anonymous>)
@rjsf/playground:     at Object.build (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:2131:14)
@rjsf/playground:     at build (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1978:51)
@rjsf/playground:     at scanImports (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41760:11)
@rjsf/playground:   The plugin "vite:dep-scan" was triggered by this import
@rjsf/playground:     src/index.jsx:4:39:
@rjsf/playground:       4 │ import { Theme as FluentUITheme } from "@rjsf/fluent-ui";
@rjsf/playground:         ╵                                        ~~~~~~~~~~~~~~~~~
@rjsf/playground: ✘ [ERROR] [plugin vite:dep-scan] Failed to resolve entry for package "@rjsf/antd". The package may have incorrect main/module/exports specified in its package.json.
@rjsf/playground:     node_modules/esbuild/lib/main.js:1365:21:
@rjsf/playground:       1365 │         let result = await callback({
@rjsf/playground:            ╵                      ^
@rjsf/playground:     at packageEntryFailure (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22004:11)
@rjsf/playground:     at resolvePackageEntry (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22001:5)
@rjsf/playground:     at tryNodeResolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21736:20)
@rjsf/playground:     at Context.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21487:28)
@rjsf/playground:     at Object.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41587:46)
@rjsf/playground:     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
@rjsf/playground:     at async resolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41815:26)
@rjsf/playground:     at async file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41989:34
@rjsf/playground:     at async requestCallbacks.on-resolve (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1365:22)
@rjsf/playground:     at async handleRequest (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:727:13)
@rjsf/playground:   This error came from the "onResolve" callback registered here:
@rjsf/playground:     node_modules/esbuild/lib/main.js:1287:20:
@rjsf/playground:       1287 │       let promise = setup({
@rjsf/playground:            ╵                     ^
@rjsf/playground:     at setup (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41979:19)
@rjsf/playground:     at handlePlugins (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1287:21)
@rjsf/playground:     at buildOrServeImpl (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:974:5)
@rjsf/playground:     at Object.buildOrServe (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:780:5)
@rjsf/playground:     at /home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:2132:17
@rjsf/playground:     at new Promise (<anonymous>)
@rjsf/playground:     at Object.build (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:2131:14)
@rjsf/playground:     at build (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1978:51)
@rjsf/playground:     at scanImports (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41760:11)
@rjsf/playground:   The plugin "vite:dep-scan" was triggered by this import
@rjsf/playground:     src/index.jsx:6:35:
@rjsf/playground:       6 │ import { Theme as AntdTheme } from "@rjsf/antd";
@rjsf/playground:         ╵                                    ~~~~~~~~~~~~
@rjsf/playground: ✘ [ERROR] [plugin vite:dep-scan] Failed to resolve entry for package "@rjsf/bootstrap-4". The package may have incorrect main/module/exports specified in its package.json.
@rjsf/playground:     node_modules/esbuild/lib/main.js:1365:21:
@rjsf/playground:       1365 │         let result = await callback({
@rjsf/playground:            ╵                      ^
@rjsf/playground:     at packageEntryFailure (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22004:11)
@rjsf/playground:     at resolvePackageEntry (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22001:5)
@rjsf/playground:     at tryNodeResolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21736:20)
@rjsf/playground:     at Context.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21487:28)
@rjsf/playground:     at Object.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41587:46)
@rjsf/playground:     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
@rjsf/playground:     at async resolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41815:26)
@rjsf/playground:     at async file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41989:34
@rjsf/playground:     at async requestCallbacks.on-resolve (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1365:22)
@rjsf/playground:     at async handleRequest (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:727:13)
@rjsf/playground:   This error came from the "onResolve" callback registered here:
@rjsf/playground:     node_modules/esbuild/lib/main.js:1287:20:
@rjsf/playground:       1287 │       let promise = setup({
@rjsf/playground:            ╵                     ^
@rjsf/playground:     at setup (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41979:19)
@rjsf/playground:     at handlePlugins (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1287:21)
@rjsf/playground:     at buildOrServeImpl (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:974:5)
@rjsf/playground:     at Object.buildOrServe (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:780:5)
@rjsf/playground:     at /home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:2132:17
@rjsf/playground:     at new Promise (<anonymous>)
@rjsf/playground:     at Object.build (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:2131:14)
@rjsf/playground:     at build (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1978:51)
@rjsf/playground:     at scanImports (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41760:11)
@rjsf/playground:   The plugin "vite:dep-scan" was triggered by this import
@rjsf/playground:     src/index.jsx:7:41:
@rjsf/playground:       7 │ import { Theme as Bootstrap4Theme } from "@rjsf/bootstrap-4";
@rjsf/playground:         ╵                                          ~~~~~~~~~~~~~~~~~~~
@rjsf/playground: ✘ [ERROR] [plugin vite:dep-scan] Failed to resolve entry for package "@rjsf/chakra-ui". The package may have incorrect main/module/exports specified in its package.json.
@rjsf/playground:     node_modules/esbuild/lib/main.js:1365:21:
@rjsf/playground:       1365 │         let result = await callback({
@rjsf/playground:            ╵                      ^
@rjsf/playground:     at packageEntryFailure (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22004:11)
@rjsf/playground:     at resolvePackageEntry (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22001:5)
@rjsf/playground:     at tryNodeResolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21736:20)
@rjsf/playground:     at Context.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21487:28)
@rjsf/playground:     at Object.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41587:46)
@rjsf/playground:     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
@rjsf/playground:     at async resolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41815:26)
@rjsf/playground:     at async file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41989:34
@rjsf/playground:     at async requestCallbacks.on-resolve (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1365:22)
@rjsf/playground:     at async handleRequest (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:727:13)
@rjsf/playground:   This error came from the "onResolve" callback registered here:
@rjsf/playground:     node_modules/esbuild/lib/main.js:1287:20:
@rjsf/playground:       1287 │       let promise = setup({
@rjsf/playground:            ╵                     ^
@rjsf/playground:     at setup (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41979:19)
@rjsf/playground:     at handlePlugins (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1287:21)
@rjsf/playground:     at buildOrServeImpl (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:974:5)
@rjsf/playground:     at Object.buildOrServe (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:780:5)
@rjsf/playground:     at /home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:2132:17
@rjsf/playground:     at new Promise (<anonymous>)
@rjsf/playground:     at Object.build (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:2131:14)
@rjsf/playground:     at build (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1978:51)
@rjsf/playground:     at scanImports (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41760:11)
@rjsf/playground:   The plugin "vite:dep-scan" was triggered by this import
@rjsf/playground:     src/index.jsx:8:39:
@rjsf/playground:       8 │ import { Theme as ChakraUITheme } from "@rjsf/chakra-ui";
@rjsf/playground:         ╵                                        ~~~~~~~~~~~~~~~~~
@rjsf/playground: ✘ [ERROR] [plugin vite:dep-scan] Failed to resolve entry for package "@rjsf/core". The package may have incorrect main/module/exports specified in its package.json.
@rjsf/playground:     node_modules/esbuild/lib/main.js:1365:21:
@rjsf/playground:       1365 │         let result = await callback({
@rjsf/playground:            ╵                      ^
@rjsf/playground:     at packageEntryFailure (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22004:11)
@rjsf/playground:     at resolvePackageEntry (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22001:5)
@rjsf/playground:     at tryNodeResolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21736:20)
@rjsf/playground:     at Context.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21487:28)
@rjsf/playground:     at Object.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41587:46)
@rjsf/playground:     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
@rjsf/playground:     at async resolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41815:26)
@rjsf/playground:     at async file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41989:34
@rjsf/playground:     at async requestCallbacks.on-resolve (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1365:22)
@rjsf/playground:     at async handleRequest (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:727:13)
@rjsf/playground:   This error came from the "onResolve" callback registered here:
@rjsf/playground:     node_modules/esbuild/lib/main.js:1287:20:
@rjsf/playground:       1287 │       let promise = setup({
@rjsf/playground:            ╵                     ^
@rjsf/playground:     at setup (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41979:19)
@rjsf/playground:     at handlePlugins (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1287:21)
@rjsf/playground:     at buildOrServeImpl (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:974:5)
@rjsf/playground:     at Object.buildOrServe (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:780:5)
@rjsf/playground:     at /home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:2132:17
@rjsf/playground:     at new Promise (<anonymous>)
@rjsf/playground:     at Object.build (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:2131:14)
@rjsf/playground:     at build (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1978:51)
@rjsf/playground:     at scanImports (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41760:11)
@rjsf/playground:   The plugin "vite:dep-scan" was triggered by this import
@rjsf/playground:     src/app.jsx:5:32:
@rjsf/playground:       5 │ import Form, { withTheme } from "@rjsf/core";
@rjsf/playground:         ╵                                 ~~~~~~~~~~~~
@rjsf/playground: ✘ [ERROR] [plugin vite:dep-scan] Failed to resolve entry for package "@rjsf/chakra-ui". The package may have incorrect main/module/exports specified in its package.json.
@rjsf/playground:     node_modules/esbuild/lib/main.js:1365:21:
@rjsf/playground:       1365 │         let result = await callback({
@rjsf/playground:            ╵                      ^
@rjsf/playground:     at packageEntryFailure (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22004:11)
@rjsf/playground:     at resolvePackageEntry (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22001:5)
@rjsf/playground:     at tryNodeResolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21736:20)
@rjsf/playground:     at Context.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21487:28)
@rjsf/playground:     at Object.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41587:46)
@rjsf/playground:     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
@rjsf/playground:     at async resolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41815:26)
@rjsf/playground:     at async file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41989:34
@rjsf/playground:     at async requestCallbacks.on-resolve (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1365:22)
@rjsf/playground:     at async handleRequest (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:727:13)
@rjsf/playground:   This error came from the "onResolve" callback registered here:
@rjsf/playground:     node_modules/esbuild/lib/main.js:1287:20:
@rjsf/playground:       1287 │       let promise = setup({
@rjsf/playground:            ╵                     ^
@rjsf/playground:     at setup (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41979:19)
@rjsf/playground:     at handlePlugins (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1287:21)
@rjsf/playground:     at buildOrServeImpl (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:974:5)
@rjsf/playground:     at Object.buildOrServe (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:780:5)
@rjsf/playground:     at /home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:2132:17
@rjsf/playground:     at new Promise (<anonymous>)
@rjsf/playground:     at Object.build (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:2131:14)
@rjsf/playground:     at build (/home/nate/react-jsonschema-form/packages/playground/node_modules/esbuild/lib/main.js:1978:51)
@rjsf/playground:     at scanImports (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41760:11)
@rjsf/playground:   The plugin "vite:dep-scan" was triggered by this import
@rjsf/playground:     src/DemoFrame.jsx:8:44:
@rjsf/playground:       8 │ import { __createChakraFrameProvider } from "@rjsf/chakra-ui";
@rjsf/playground:         ╵                                             ~~~~~~~~~~~~~~~~~
@rjsf/playground: Build failed with 8 errors:
@rjsf/playground: node_modules/esbuild/lib/main.js:1365:21: ERROR: [plugin: vite:dep-scan] Failed to resolve entry for package "@rjsf/antd". The package may have incorrect main/module/exports specified in its package.json.
@rjsf/playground: node_modules/esbuild/lib/main.js:1365:21: ERROR: [plugin: vite:dep-scan] Failed to resolve entry for package "@rjsf/bootstrap-4". The package may have incorrect main/module/exports specified in its package.json.
@rjsf/playground: node_modules/esbuild/lib/main.js:1365:21: ERROR: [plugin: vite:dep-scan] Failed to resolve entry for package "@rjsf/chakra-ui". The package may have incorrect main/module/exports specified in its package.json.
@rjsf/playground: node_modules/esbuild/lib/main.js:1365:21: ERROR: [plugin: vite:dep-scan] Failed to resolve entry for package "@rjsf/chakra-ui". The package may have incorrect main/module/exports specified in its package.json.
@rjsf/playground: node_modules/esbuild/lib/main.js:1365:21: ERROR: [plugin: vite:dep-scan] Failed to resolve entry for package "@rjsf/core". The package may have incorrect main/module/exports specified in its package.json.
@rjsf/playground: ...
@rjsf/core: - Compiling modules...
@rjsf/antd: - Compiling modules...
@rjsf/bootstrap-4: - Compiling modules...
@rjsf/chakra-ui: - Compiling modules...
@rjsf/material-ui: - Compiling modules...
@rjsf/fluent-ui: - Compiling modules...
@rjsf/mui: - Compiling modules...
docs: [INFO] Starting the development server...
@rjsf/playground: 5:13:17 PM [vite] Internal server error: Failed to resolve entry for package "@rjsf/material-ui". The package may have incorrect main/module/exports specified in its package.json.
@rjsf/playground:   Plugin: vite:import-analysis
@rjsf/playground:   File: /home/nate/react-jsonschema-form/packages/playground/src/index.jsx
@rjsf/playground:       at packageEntryFailure (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22004:11)
@rjsf/playground:       at resolvePackageEntry (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22001:5)
@rjsf/playground:       at tryNodeResolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21736:20)
@rjsf/playground:       at Context.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21487:28)
@rjsf/playground:       at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
@rjsf/playground:       at async Object.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41587:32)
@rjsf/playground:       at async TransformContext.resolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41334:23)
@rjsf/playground:       at async normalizeUrl (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:39685:34)
@rjsf/playground:       at async TransformContext.transform (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:39826:47)
@rjsf/playground:       at async Object.transform (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41660:30)
@rjsf/playground:       at async loadAndTransform (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:39466:29)
@rjsf/playground: 5:13:17 PM [vite] Internal server error: Failed to resolve entry for package "@rjsf/material-ui". The package may have incorrect main/module/exports specified in its package.json.
@rjsf/playground:   Plugin: vite:import-analysis
@rjsf/playground:   File: /home/nate/react-jsonschema-form/packages/playground/src/index.jsx
@rjsf/playground:       at packageEntryFailure (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22004:11)
@rjsf/playground:       at resolvePackageEntry (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22001:5)
@rjsf/playground:       at tryNodeResolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21736:20)
@rjsf/playground:       at Context.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21487:28)
@rjsf/playground:       at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
@rjsf/playground:       at async Object.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41587:32)
@rjsf/playground:       at async TransformContext.resolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41334:23)
@rjsf/playground:       at async normalizeUrl (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:39685:34)
@rjsf/playground:       at async TransformContext.transform (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:39826:47)
@rjsf/playground:       at async Object.transform (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41660:30)
@rjsf/playground:       at async loadAndTransform (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:39466:29)
@rjsf/playground: 5:13:17 PM [vite] Internal server error: Failed to resolve entry for package "@rjsf/material-ui". The package may have incorrect main/module/exports specified in its package.json.
@rjsf/playground:   Plugin: vite:import-analysis
@rjsf/playground:   File: /home/nate/react-jsonschema-form/packages/playground/src/index.jsx
@rjsf/playground:       at packageEntryFailure (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22004:11)
@rjsf/playground:       at resolvePackageEntry (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:22001:5)
@rjsf/playground:       at tryNodeResolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21736:20)
@rjsf/playground:       at Context.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:21487:28)
@rjsf/playground:       at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
@rjsf/playground:       at async Object.resolveId (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41587:32)
@rjsf/playground:       at async TransformContext.resolve (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41334:23)
@rjsf/playground:       at async normalizeUrl (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:39685:34)
@rjsf/playground:       at async TransformContext.transform (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:39826:47)
@rjsf/playground:       at async Object.transform (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:41660:30)
@rjsf/playground:       at async loadAndTransform (file:///home/nate/react-jsonschema-form/packages/playground/node_modules/vite/dist/node/chunks/dep-3007b26d.js:39466:29)
docs: [SUCCESS] Docusaurus website is running at: http://localhost:3000/react-jsonschema-form/docs/
docs: ℹ Compiling Client
@rjsf/antd: ✖ Failed to compile
@rjsf/antd: (typescript) Error: /home/nate/react-jsonschema-form/packages/antd/src/index.ts(3,50): semantic error TS7016: Could not find a declaration file for module '@rjsf/core'. '/home/nate/react-jsonschema-form/packages/antd/node_modules/@rjsf/core/dist/index.js' implicitly has an 'any' type.
@rjsf/antd:   Try `npm i --save-dev @types/rjsf__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@rjsf/core';`
@rjsf/antd: Error: /home/nate/react-jsonschema-form/packages/antd/src/index.ts(3,50): semantic error TS7016: Could not find a declaration file for module '@rjsf/core'. '/home/nate/react-jsonschema-form/packages/antd/node_modules/@rjsf/core/dist/index.js' implicitly has an 'any' type.
@rjsf/antd:   Try `npm i --save-dev @types/rjsf__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@rjsf/core';`
@rjsf/antd:     at error (/home/nate/react-jsonschema-form/packages/antd/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:198:30)
@rjsf/antd:     at throwPluginError (/home/nate/react-jsonschema-form/packages/antd/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:21894:12)
@rjsf/antd:     at Object.error (/home/nate/react-jsonschema-form/packages/antd/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:22616:20)
@rjsf/antd:     at Object.error (/home/nate/react-jsonschema-form/packages/antd/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:22071:42)
@rjsf/antd:     at RollupContext.error (/home/nate/react-jsonschema-form/packages/antd/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:20259:30)
@rjsf/antd:     at /home/nate/react-jsonschema-form/packages/antd/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:29527:23
@rjsf/antd:     at Array.forEach (<anonymous>)
@rjsf/antd:     at printDiagnostics (/home/nate/react-jsonschema-form/packages/antd/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:29500:17)
@rjsf/antd:     at Object.transform (/home/nate/react-jsonschema-form/packages/antd/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:29785:17)
@rjsf/antd:     at /home/nate/react-jsonschema-form/packages/antd/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:22825:37
@rjsf/bootstrap-4: ✖ Failed to compile
@rjsf/bootstrap-4: (typescript) Error: /home/nate/react-jsonschema-form/packages/bootstrap-4/src/Form/Form.tsx(3,38): semantic error TS7016: Could not find a declaration file for module '@rjsf/core'. '/home/nate/react-jsonschema-form/packages/bootstrap-4/node_modules/@rjsf/core/dist/index.js' implicitly has an 'any' type.
@rjsf/bootstrap-4:   Try `npm i --save-dev @types/rjsf__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@rjsf/core';`
@rjsf/bootstrap-4: Error: /home/nate/react-jsonschema-form/packages/bootstrap-4/src/Form/Form.tsx(3,38): semantic error TS7016: Could not find a declaration file for module '@rjsf/core'. '/home/nate/react-jsonschema-form/packages/bootstrap-4/node_modules/@rjsf/core/dist/index.js' implicitly has an 'any' type.
@rjsf/bootstrap-4:   Try `npm i --save-dev @types/rjsf__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@rjsf/core';`
@rjsf/bootstrap-4:     at error (/home/nate/react-jsonschema-form/packages/bootstrap-4/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:198:30)
@rjsf/bootstrap-4:     at throwPluginError (/home/nate/react-jsonschema-form/packages/bootstrap-4/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:21894:12)
@rjsf/bootstrap-4:     at Object.error (/home/nate/react-jsonschema-form/packages/bootstrap-4/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:22616:20)
@rjsf/bootstrap-4:     at Object.error (/home/nate/react-jsonschema-form/packages/bootstrap-4/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:22071:42)
@rjsf/bootstrap-4:     at RollupContext.error (/home/nate/react-jsonschema-form/packages/bootstrap-4/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:20259:30)
@rjsf/bootstrap-4:     at /home/nate/react-jsonschema-form/packages/bootstrap-4/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:29527:23
@rjsf/bootstrap-4:     at Array.forEach (<anonymous>)
@rjsf/bootstrap-4:     at printDiagnostics (/home/nate/react-jsonschema-form/packages/bootstrap-4/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:29500:17)
@rjsf/bootstrap-4:     at Object.transform (/home/nate/react-jsonschema-form/packages/bootstrap-4/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:29785:17)
@rjsf/bootstrap-4:     at /home/nate/react-jsonschema-form/packages/bootstrap-4/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:22825:37
@rjsf/material-ui: ✖ Failed to compile
@rjsf/material-ui: (typescript) Error: /home/nate/react-jsonschema-form/packages/material-ui/src/MuiForm/MuiForm.tsx(2,38): semantic error TS7016: Could not find a declaration file for module '@rjsf/core'. '/home/nate/react-jsonschema-form/packages/material-ui/node_modules/@rjsf/core/dist/index.js' implicitly has an 'any' type.
@rjsf/material-ui:   Try `npm i --save-dev @types/rjsf__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@rjsf/core';`
@rjsf/material-ui: Error: /home/nate/react-jsonschema-form/packages/material-ui/src/MuiForm/MuiForm.tsx(2,38): semantic error TS7016: Could not find a declaration file for module '@rjsf/core'. '/home/nate/react-jsonschema-form/packages/material-ui/node_modules/@rjsf/core/dist/index.js' implicitly has an 'any' type.
@rjsf/material-ui:   Try `npm i --save-dev @types/rjsf__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@rjsf/core';`
@rjsf/material-ui:     at error (/home/nate/react-jsonschema-form/packages/material-ui/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:198:30)
@rjsf/material-ui:     at throwPluginError (/home/nate/react-jsonschema-form/packages/material-ui/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:21894:12)
@rjsf/material-ui:     at Object.error (/home/nate/react-jsonschema-form/packages/material-ui/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:22616:20)
@rjsf/material-ui:     at Object.error (/home/nate/react-jsonschema-form/packages/material-ui/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:22071:42)
@rjsf/material-ui:     at RollupContext.error (/home/nate/react-jsonschema-form/packages/material-ui/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:20259:30)
@rjsf/material-ui:     at /home/nate/react-jsonschema-form/packages/material-ui/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:29527:23
@rjsf/material-ui:     at Array.forEach (<anonymous>)
@rjsf/material-ui:     at printDiagnostics (/home/nate/react-jsonschema-form/packages/material-ui/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:29500:17)
@rjsf/material-ui:     at Object.transform (/home/nate/react-jsonschema-form/packages/material-ui/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:29785:17)
@rjsf/material-ui:     at /home/nate/react-jsonschema-form/packages/material-ui/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:22825:37
@rjsf/mui: ✖ Failed to compile
@rjsf/mui: (typescript) Error: /home/nate/react-jsonschema-form/packages/mui/src/MuiForm/MuiForm.tsx(2,38): semantic error TS7016: Could not find a declaration file for module '@rjsf/core'. '/home/nate/react-jsonschema-form/packages/mui/node_modules/@rjsf/core/dist/index.js' implicitly has an 'any' type.
@rjsf/mui:   Try `npm i --save-dev @types/rjsf__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@rjsf/core';`
@rjsf/mui: Error: /home/nate/react-jsonschema-form/packages/mui/src/MuiForm/MuiForm.tsx(2,38): semantic error TS7016: Could not find a declaration file for module '@rjsf/core'. '/home/nate/react-jsonschema-form/packages/mui/node_modules/@rjsf/core/dist/index.js' implicitly has an 'any' type.
@rjsf/mui:   Try `npm i --save-dev @types/rjsf__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@rjsf/core';`
@rjsf/mui:     at error (/home/nate/react-jsonschema-form/packages/mui/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:198:30)
@rjsf/mui:     at throwPluginError (/home/nate/react-jsonschema-form/packages/mui/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:21894:12)
@rjsf/mui:     at Object.error (/home/nate/react-jsonschema-form/packages/mui/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:22616:20)
@rjsf/mui:     at Object.error (/home/nate/react-jsonschema-form/packages/mui/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:22071:42)
@rjsf/mui:     at RollupContext.error (/home/nate/react-jsonschema-form/packages/mui/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:20259:30)
@rjsf/mui:     at /home/nate/react-jsonschema-form/packages/mui/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:29527:23
@rjsf/mui:     at Array.forEach (<anonymous>)
@rjsf/mui:     at printDiagnostics (/home/nate/react-jsonschema-form/packages/mui/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:29500:17)
@rjsf/mui:     at Object.transform (/home/nate/react-jsonschema-form/packages/mui/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:29785:17)
@rjsf/mui:     at /home/nate/react-jsonschema-form/packages/mui/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:22825:37
@rjsf/fluent-ui: ✖ Failed to compile
@rjsf/fluent-ui: (typescript) Error: /home/nate/react-jsonschema-form/packages/fluent-ui/src/FuiForm/FuiForm.ts(2,38): semantic error TS7016: Could not find a declaration file for module '@rjsf/core'. '/home/nate/react-jsonschema-form/packages/fluent-ui/node_modules/@rjsf/core/dist/index.js' implicitly has an 'any' type.
@rjsf/fluent-ui:   Try `npm i --save-dev @types/rjsf__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@rjsf/core';`
@rjsf/fluent-ui: Error: /home/nate/react-jsonschema-form/packages/fluent-ui/src/FuiForm/FuiForm.ts(2,38): semantic error TS7016: Could not find a declaration file for module '@rjsf/core'. '/home/nate/react-jsonschema-form/packages/fluent-ui/node_modules/@rjsf/core/dist/index.js' implicitly has an 'any' type.
@rjsf/fluent-ui:   Try `npm i --save-dev @types/rjsf__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@rjsf/core';`
@rjsf/fluent-ui:     at error (/home/nate/react-jsonschema-form/packages/fluent-ui/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:198:30)
@rjsf/fluent-ui:     at throwPluginError (/home/nate/react-jsonschema-form/packages/fluent-ui/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:21894:12)
@rjsf/fluent-ui:     at Object.error (/home/nate/react-jsonschema-form/packages/fluent-ui/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:22616:20)
@rjsf/fluent-ui:     at Object.error (/home/nate/react-jsonschema-form/packages/fluent-ui/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:22071:42)
@rjsf/fluent-ui:     at RollupContext.error (/home/nate/react-jsonschema-form/packages/fluent-ui/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:20259:30)
@rjsf/fluent-ui:     at /home/nate/react-jsonschema-form/packages/fluent-ui/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:29527:23
@rjsf/fluent-ui:     at Array.forEach (<anonymous>)
@rjsf/fluent-ui:     at printDiagnostics (/home/nate/react-jsonschema-form/packages/fluent-ui/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:29500:17)
@rjsf/fluent-ui:     at Object.transform (/home/nate/react-jsonschema-form/packages/fluent-ui/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:29785:17)
@rjsf/fluent-ui:     at /home/nate/react-jsonschema-form/packages/fluent-ui/node_modules/dts-cli/node_modules/rollup/dist/shared/rollup.js:22825:37
@rjsf/chakra-ui: ✖ Failed to compile
@rjsf/chakra-ui: (typescript) Error: /home/nate/react-jsonschema-form/packages/chakra-ui/src/Form/Form.tsx(2,38): semantic error TS7016: Could not find a declaration file for module '@rjsf/core'. '/home/nate/react-jsonschema-form/packages/chakra-ui/node_modules/@rjsf/core/dist/index.js' implicitly has an 'any' type.
@rjsf/chakra-ui:   Try `npm i --save-dev @types/rjsf__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@rjsf/core';`
@rjsf/chakra-ui: Error: /home/nate/react-jsonschema-form/packages/chakra-ui/src/Form/Form.tsx(2,38): semantic error TS7016: Could not find a declaration file for module '@rjsf/core'. '/home/nate/react-jsonschema-form/packages/chakra-ui/node_modules/@rjsf/core/dist/index.js' implicitly has an 'any' type.
@rjsf/chakra-ui:   Try `npm i --save-dev @types/rjsf__core` if it exists or add a new declaration (.d.ts) file containing `declare module '@rjsf/core';`
@rjsf/chakra-ui:     at error (/home/nate/react-jsonschema-form/packages/chakra-ui/node_modules/rollup/dist/shared/rollup.js:198:30)
@rjsf/chakra-ui:     at throwPluginError (/home/nate/react-jsonschema-form/packages/chakra-ui/node_modules/rollup/dist/shared/rollup.js:21894:12)
@rjsf/chakra-ui:     at Object.error (/home/nate/react-jsonschema-form/packages/chakra-ui/node_modules/rollup/dist/shared/rollup.js:22616:20)
@rjsf/chakra-ui:     at Object.error (/home/nate/react-jsonschema-form/packages/chakra-ui/node_modules/rollup/dist/shared/rollup.js:22071:42)
@rjsf/chakra-ui:     at RollupContext.error (/home/nate/react-jsonschema-form/packages/chakra-ui/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:20259:30)
@rjsf/chakra-ui:     at /home/nate/react-jsonschema-form/packages/chakra-ui/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:29527:23
@rjsf/chakra-ui:     at Array.forEach (<anonymous>)
@rjsf/chakra-ui:     at printDiagnostics (/home/nate/react-jsonschema-form/packages/chakra-ui/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:29500:17)
@rjsf/chakra-ui:     at Object.transform (/home/nate/react-jsonschema-form/packages/chakra-ui/node_modules/dts-cli/node_modules/rollup-plugin-typescript2/dist/rollup-plugin-typescript2.cjs.js:29785:17)
@rjsf/chakra-ui:     at /home/nate/react-jsonschema-form/packages/chakra-ui/node_modules/rollup/dist/shared/rollup.js:22825:37
@rjsf/chakra-ui:     at runNextTicks (node:internal/process/task_queues:60:5)
@rjsf/chakra-ui:     at process.processImmediate (node:internal/timers:442:9)
^C
```

### Expected Behavior

Running `npm start` launches the playground locally and enables hot-reload without error.

### Steps To Reproduce

1. Clone repository
2. Run `npm run build`
3. Run `npm start`
4. See the errors in the console and in Firefox see the Vite error

### Environment

```markdown
- OS: Ubuntu 22.04 (WSL2 on Windows 11)
- Node: 19.4.0
- npm: 9.3.0
```


### Anything else?

_No response_

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -22,6 +22,25 @@ should change the heading of the (upcoming) version to include a major version b
 
 - Updated `AltDateWidget` to remove an infinite loop caused by two conflicting effects by merging them with additional checking of original `value` against the current value, fixing [#3516](https://github.com/rjsf-team/react-jsonschema-form/issues/3516)
 
+## @rjsf/utils
+
+- Updated the documentation of `getTestValidator()` and the `schema.test.ts` file to help developers
+
+## @rjsf/validator-ajv6
+
+- Updated the documentation of `getTestValidator()` and the `schema.test.ts` file to help developers
+
+## @rjsf/validator-ajv8
+
+- Updated the documentation of `getTestValidator()` and the `schema.test.ts` file to help developers
+
+## Dev / docs / playground
+
+- Updated the `internals` documentation to use a React ref in the example, fixing [#3520](https://github.com/rjsf-team/react-jsonschema-form/issues/3520)
+- Updated the `contributing` documentation to describe the new development process needed for a `Vite` playground, fixing [#3478](https://github.com/rjsf-team/react-jsonschema-form/issues/3478)
+  - Also fixed the `package.json` files to remove `npm start` in the subdirectories and change the root one to describe the new process
+- Updated the `semantic-ui/uiSchema` documentation to switch the default for `horizontalButtons` to be true per changes made in `5.3.0`
+
 # 5.3.0
 
 ## @rjsf/antd

--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
     "cs-format": "lerna run cs-format",
     "build": "lerna run --stream build",
     "build-serial": "lerna run --concurrency 1 --stream build",
-    "start": "lerna run start --stream --parallel",
+    "start": "echo 'use \"npm run build\" from main directory and then \"npm start\" in the playground package'",
     "pre-commit:husky": "lerna run --concurrency 1 --stream precommit",
     "prepare": "husky install",
     "format": "prettier --write .",

--- a/packages/antd/package.json
+++ b/packages/antd/package.json
@@ -6,7 +6,6 @@
   "module": "dist/antd.esm.js",
   "typings": "dist/index.d.ts",
   "scripts": {
-    "start": "dts watch",
     "build": "rimraf dist && dts build --rollupTypes --format cjs,esm,umd",
     "cs-check": "prettier -l \"{src,test}/**/*.ts?(x)\"",
     "cs-format": "prettier \"{src,test}/**/*.ts?(x)\" --write",

--- a/packages/bootstrap-4/package.json
+++ b/packages/bootstrap-4/package.json
@@ -13,7 +13,6 @@
     "url": ""
   },
   "scripts": {
-    "start": "dts watch",
     "build": "rimraf dist && dts build --rollupTypes --format cjs,esm,umd",
     "cs-check": "prettier -l \"{src,test}/**/*.ts?(x)\"",
     "cs-format": "prettier \"{src,test}/**/*.ts?(x)\" --write",

--- a/packages/chakra-ui/package.json
+++ b/packages/chakra-ui/package.json
@@ -9,7 +9,6 @@
     "dist"
   ],
   "scripts": {
-    "start": "dts watch",
     "build": "rimraf dist && dts build --rollupTypes --format cjs,esm,umd",
     "cs-check": "prettier -l \"{src,test}/**/*.ts?(x)\"",
     "cs-format": "prettier \"{src,test}/**/*.ts?(x)\" --write",

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -9,7 +9,6 @@
     "lint": "eslint src test",
     "precommit": "lint-staged",
     "publish-to-npm": "npm run build && npm publish",
-    "start": "dts watch",
     "test": "dts test",
     "test:debug": "node --inspect-brk node_modules/.bin/dts test",
     "test:watch": "dts test --watch",

--- a/packages/docs/docs/advanced-customization/internals.md
+++ b/packages/docs/docs/advanced-customization/internals.md
@@ -79,7 +79,9 @@ You can use the reference to get your `Form` component and call the `submit` met
 This method will dispatch the `submit` event of the form, and the function, that is passed to `onSubmit` props, will be called.
 
 ```tsx
+import { createRef } from 'react';
 import { RJSFSchema, UiSchema } from '@rjsf/utils';
+import { Form } from '@rjsf/core';
 import validator from '@rjsf/validator-ajv8';
 
 const onSubmit = ({ formData }) => console.log('Data submitted: ', formData);
@@ -89,17 +91,17 @@ const schema: RJSFSchema = {
   type: 'string',
 };
 
+const formRef = createRef<Form>();
+
 render(
   <Form
     schema={schema}
     validator={validator}
     onSubmit={onSubmit}
-    ref={(form) => {
-      yourForm = form;
-    }}
+    ref={formRef}
   />,
   document.getElementById('app')
 );
 
-yourForm.submit();
+formRef.current.submit();
 ```

--- a/packages/docs/docs/api-reference/themes/semantic-ui/uiSchema.md
+++ b/packages/docs/docs/api-reference/themes/semantic-ui/uiSchema.md
@@ -71,7 +71,7 @@ To specify a uiSchema that applies to array items, specify the semantic uiSchema
 
 ```
 wrapItem: wrap each array item in a Segment
-horizontalButtons: horizontal buttons instead of the default vertical
+horizontalButtons: vertical buttons instead of the default horizontal
 ```
 
 ```tsx
@@ -89,7 +89,7 @@ const uiSchema: UiSchema = {
   'ui:options': {
     semantic: {
       wrapItem: true,
-      horizontalButtons: true,
+      horizontalButtons: false,
     },
   },
 };

--- a/packages/docs/docs/contributing.md
+++ b/packages/docs/docs/contributing.md
@@ -2,15 +2,21 @@
 
 ## Development server
 
-When developing, run the following from the root-level directory:
+We are using `Vite` to power our `playground`, which caches all the built `@rjsf/*` distributions.
+In order to test the `playground` locally after a new clone or fresh pull from `main`, run the following from the root directory of the monorepo:
 
 ```bash
 npm install
 npm run build
+cd packages/playground
 npm start
 ```
 
-All packages will be live-built, and a live development server showcasing components with hot reload enabled will then run at [localhost:8080](http://localhost:8080).
+This will start the live development server showcasing components at [localhost:8080](http://localhost:8080).
+
+Whenever you make changes to source code, stop the running playground and return to the root directory and rerun `npm run build`.
+Thanks to `nx` caching, this should only rebuild what is necessary.
+After the build is complete, return to the root of the `playground` and restart the server via `npm start`.
 
 ### First time step
 
@@ -32,6 +38,8 @@ When building in environments with limited memory, such as Netlify, it's recomme
 All the JavaScript/Typescript code in this project conforms to the [prettier](https://github.com/prettier/prettier) coding style.
 Code is automatically prettified upon commit using precommit hooks, assuming you followed the `First time step` above.
 
+You can also run `npm cs-format` within any package directory you are changing.
+
 ## Documentation
 
 We use [Docusaurus](https://docusaurus.io/) to build our documentation. To run documentation locally, run:
@@ -45,8 +53,10 @@ Documentation will be served on [localhost:3000](http://localhost:3000).
 
 ## Tests
 
+You can run all tests from the root directory OR from `packages` subdirectory using the following command:
+
 ```bash
-npm test
+npm run test
 ```
 
 ### Code coverage
@@ -59,6 +69,8 @@ The full report can be seen by opening `./coverage/lcov-report/index.html`.
 
 100% code coverage is required by the `@rjsf/utils` and `@rjsf/validator-ajv6` and `@rjsf/validator-ajv8` tests.
 If you make changes to those libraries, you will have to maintain that coverage, otherwise the tests will fail.
+
+> NOTE: All three of these directories share the same tests for verifying `validator` based APIs. See the documentation in the `getTestValidator()` functions for more information. 
 
 ## Releasing
 

--- a/packages/docs/package.json
+++ b/packages/docs/package.json
@@ -12,7 +12,18 @@
     "serve": "docusaurus serve",
     "write-translations": "docusaurus write-translations",
     "write-heading-ids": "docusaurus write-heading-ids",
-    "typecheck": "tsc"
+    "typecheck": "tsc",
+    "precommit": "lint-staged",
+    "cs-check": "prettier -l \"{src, docs}/**/*.(ts?(x)|md|css)\"",
+    "cs-format": "prettier \"{src, docs}/**/*.(ts?(x)|md|css)\" --write"
+  },
+  "lint-staged": {
+    "src/**/*.(ts?(x)|css)": [
+      "prettier --write"
+    ],
+    "docs/**/*.md": [
+      "prettier --write"
+    ]
   },
   "dependencies": {
     "@docusaurus/core": "^2.3.1",

--- a/packages/fluent-ui/package.json
+++ b/packages/fluent-ui/package.json
@@ -9,7 +9,6 @@
     "dist"
   ],
   "scripts": {
-    "start": "dts watch",
     "build": "rimraf dist && dts build --rollupTypes --format cjs,esm,umd",
     "cs-check": "prettier -l \"{src,test}/**/*.ts?(x)\"",
     "cs-format": "prettier \"{src,test}/**/*.ts?(x)\" --write",

--- a/packages/material-ui/package.json
+++ b/packages/material-ui/package.json
@@ -9,7 +9,6 @@
     "dist"
   ],
   "scripts": {
-    "start": "dts watch",
     "build": "rimraf dist && dts build --rollupTypes --format cjs,esm,umd",
     "cs-check": "prettier -l \"{src,test}/**/*.ts?(x)\"",
     "cs-format": "prettier \"{src,test}/**/*.ts?(x)\" --write",

--- a/packages/mui/package.json
+++ b/packages/mui/package.json
@@ -9,7 +9,6 @@
     "dist"
   ],
   "scripts": {
-    "start": "dts watch",
     "build": "rimraf dist && dts build --rollupTypes --format cjs,esm,umd",
     "cs-check": "prettier -l \"{src,test}/**/*.ts?(x)\"",
     "cs-format": "prettier \"{src,test}/**/*.ts?(x)\" --write",

--- a/packages/playground/package.json
+++ b/packages/playground/package.json
@@ -8,8 +8,8 @@
     "build:lib": "rimraf lib && cross-env NODE_ENV=production babel -d lib/ src/",
     "prepublishOnly": "npm run build:lib",
     "build:dist": "rimraf dist && cross-env NODE_ENV=production vite build",
-    "cs-check": "prettier -l \"{playground,src,test}/**/*.js\"",
-    "cs-format": "prettier \"{playground,src,test}/**/*.js\" --write",
+    "cs-check": "prettier -l \"src/**/*.js\"",
+    "cs-format": "prettier \"src/**/*.js\" --write",
     "build": "rimraf build && cross-env NODE_ENV=production vite build",
     "lint": "eslint src",
     "precommit": "lint-staged",
@@ -19,7 +19,7 @@
     "preview": "vite preview"
   },
   "lint-staged": {
-    "{src}/**/*.js": [
+    "src/**/*.js": [
       "eslint --fix",
       "prettier --write"
     ]

--- a/packages/utils/test/schema.test.ts
+++ b/packages/utils/test/schema.test.ts
@@ -16,6 +16,7 @@ import {
 
 const testValidator = getTestValidator({});
 
+// NOTE: to restrict which tests to run, you can temporarily comment out any tests you aren't needing
 getDefaultFormStateTest(testValidator);
 getDisplayLabelTest(testValidator);
 getClosestMatchingOptionTest(testValidator);

--- a/packages/utils/test/testUtils/getTestValidator.ts
+++ b/packages/utils/test/testUtils/getTestValidator.ts
@@ -1,9 +1,19 @@
 import { RJSFValidationError, ValidationData } from '../../src';
 import { TestValidatorParams, TestValidatorType } from '../schema/types';
 
+/** A test validator implements the `ValidatorType` interface needed by all the `schema` based tests. Inside the `utils`
+ * directory, there is no actual validator implementation, so it can be necessary to mock the expected return values
+ * of the `validateFormData()`, `isValid()` and `errorList()` APIs to imitate what is expected from the real validator
+ * implementations. The `setReturnValue()` function is used to set up this mocked data. It may take a some investigation
+ * of the actual `validator` implementations to understand how to set the mocks when writing or updating tests. Feel
+ * free to run the validator tests first if you aren't sure whether your mocks are correct.
+ *
+ * All tests that use the `TestValidator` are contained within the `@rjsf/utils/test/schema` directory and are run via
+ * the `schema.test.ts` file.
+ */
 export default function getTestValidator<T = any>({
-  isValid = [],
   data = [],
+  isValid = [],
   errorList = [],
 }: TestValidatorParams): TestValidatorType {
   const testValidator: {

--- a/packages/validator-ajv6/test/utilsTests/getTestValidator.ts
+++ b/packages/validator-ajv6/test/utilsTests/getTestValidator.ts
@@ -10,6 +10,12 @@ import {
 import { TestValidatorType } from '@rjsf/utils/test/schema';
 import { customizeValidator, CustomValidatorOptionsType } from '../../src';
 
+/** In this environment, a test validator merely creates an internal `AJV6` validator with the custom `options` and
+ * then forwards all calls into it. It is provided to mirror the test validator in the `utils` directory, except that
+ * the `setReturnValues()` function does nothing.
+ *
+ * @param options
+ */
 export default function getTestValidator<T = any>(options: CustomValidatorOptionsType): TestValidatorType {
   const validator = customizeValidator<T>(options);
   return {

--- a/packages/validator-ajv6/test/utilsTests/schema.test.ts
+++ b/packages/validator-ajv6/test/utilsTests/schema.test.ts
@@ -17,6 +17,7 @@ import getTestValidator from './getTestValidator';
 
 const testValidator = getTestValidator({});
 
+// NOTE: to restrict which tests to run, you can temporarily comment out any tests you aren't needing
 getDefaultFormStateTest(testValidator);
 getDisplayLabelTest(testValidator);
 getClosestMatchingOptionTest(testValidator);

--- a/packages/validator-ajv8/test/utilsTests/getTestValidator.ts
+++ b/packages/validator-ajv8/test/utilsTests/getTestValidator.ts
@@ -10,6 +10,12 @@ import {
 import { TestValidatorType } from '@rjsf/utils/test/schema';
 import { customizeValidator, CustomValidatorOptionsType } from '../../src';
 
+/** In this environment, a test validator merely creates an internal `AJV8` validator with the custom `options` and
+ * then forwards all calls into it. It is provided to mirror the test validator in the `utils` directory, except that
+ * the `setReturnValues()` function does nothing.
+ *
+ * @param options
+ */
 export default function getTestValidator<T = any>(options: CustomValidatorOptionsType): TestValidatorType {
   const validator = customizeValidator<T>(options);
   return {

--- a/packages/validator-ajv8/test/utilsTests/schema.test.ts
+++ b/packages/validator-ajv8/test/utilsTests/schema.test.ts
@@ -19,6 +19,7 @@ import getTestValidator from './getTestValidator';
 
 const testValidator = getTestValidator({});
 
+// NOTE: to restrict which tests to run, you can temporarily comment out any tests you aren't needing
 getDefaultFormStateTest(testValidator);
 getDisplayLabelTest(testValidator);
 getClosestMatchingOptionTest(testValidator);
@@ -34,6 +35,7 @@ toPathSchemaTest(testValidator);
 
 const testValidator2019 = getTestValidator({ AjvClass: Ajv2019 });
 
+// NOTE: to restrict which tests to run, you can temporarily comment out any tests you aren't needing
 getDefaultFormStateTest(testValidator2019);
 getDisplayLabelTest(testValidator2019);
 getClosestMatchingOptionTest(testValidator2019);
@@ -49,6 +51,7 @@ toPathSchemaTest(testValidator2019);
 
 const testValidator2020 = getTestValidator({ AjvClass: Ajv2020 });
 
+// NOTE: to restrict which tests to run, you can temporarily comment out any tests you aren't needing
 getDefaultFormStateTest(testValidator2020);
 getDisplayLabelTest(testValidator2020);
 getClosestMatchingOptionTest(testValidator2020);

